Dokumentationen/test/test_cases.adoc

```diff
@@ -39,13 +39,57 @@ a| * Datum und Uhrzeit, Studiengang
 ==== TC04 - Keine Mehrfachauswahl eines Experiments pro Buchung möglich
 
 ==== TC05 - Abbruch muss möglich sein
+|===
+
+|*Beschreibung* |Der Dozent beginnt einen Buchungsprozess, entscheidet sich aber zu einem beliebigen Zeitpunkt vor Buchungsabschluss dazu, diesen abzubrechen. Erwartet wird, dass ein Buchungsabbruch jederzeit fehlerfrei möglich ist.  
+
+
+|*Vorbedingungen*
+a| * Dozent ist angemeldet  
+* Buchung wurde gestartet 
+
+|*Nachbedingung*
+a| * Buchung wurde abgebrochen und es kam zu keinem Datenbankeintrag 
+* Dozent befindet sich auf der Seite, auf der er die Buchung begonnen hat 
+
+|*Benötigte Daten*
+a| * Zugangsdaten eines Dozentenkontos  
+
+|===
 
 === Test Cases für Use Case "Buchungsjournal verwalten"
 
 ==== TC06 - Journal enthält chronologisch geordnete Buchungen
+|===
+
+|*Beschreibung* |Der Dozent will seine Buchungen im Journal einsehen. Erwartet wird, dass die Buchungsauflistung chronologisch geordnet (bezüglich der Datums- und Zeitangaben (und der Reihenfolge bei mehreren Buchungen pro Vorlesung)) und inhaltlich richtig ist. 
+
 
+|*Vorbedingungen*
+a| * Dozent ist angemeldet  
+* Buchungen wurden erfolgreich getätigt
+|*Nachbedingung*
+a| * Dozent befindet sich auf der Journalseite 
+
+|*Benötigte Daten*
+a| * Zugangsdaten eines Dozentenkontos  
+* vollständige und valide Buchungseinträge in der Datenbank 
```

Comment:
unvollständige und invalide Daten sollten ja gar nicht  gespeichert werden können also so mit ist der Punkt ein bisschen zu viel oder ? und wir brauchen ja auch nicht mehrere 
-> mindesten ein Buchungseintrag im System 
Und das würde ich auch in die Vorbedingung schreiben weil, das ein Experiment benötigt wird haben wir auch da reingeschrieben

Reply:
Bezüglich der Vollständigkeit und Validität stimme ich dir zu. 
Ich denke jedoch, wir brauchen schon mehrere Buchungen, weil es mit einer einzigen schwer wird eine chronologische Korrektheit nachzuweisen. Ich wollte mich da jetzt nicht zu sehr festlegen aber ich denke sogar, dass wir mind. 4 Buchungssätze brauchen : 
1. einen mit separatem Datum und Uhrzeit
2. zwei (mit dem selben Datum aber unterschiedlicher Uhrzeit)
3. einen (mit mehreren Experimenten in einer Vorlesung) 

um darauf zu testen, dass auch wirklich alles chronologisch richtig geordnet wird (Datum, Uhrzeit, Reihenfolge).
Ist das verständlich?  (ähnliches Prinzip wie beim TC01 - je mehr desto besser) 

Zur Vorbedingung habe ich ja "Buchungen wurden erfolgreich getätigt geschrieben". Ergänzend dazu sollten dann die Buchungsdaten als notwendig vermerkt werden.

Reply:
Ja hast recht, habe ich nicht bedacht :D